pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -274,6 +275,20 @@ type BindingSpec struct {
 	// Names of subordinate objects to create
 	SecretName string `json:"secretName"`
 
+	// TODO: replace with PodPresetTemplateSpec
+
+	// The name of a PodPreset to create.  Must be specified with
```

Comment:
Yeah, the error message below led me to believe only one needs to be set. But it looks like the error message itself is incorrect. See below.